backend-rust/src/graphql_api/token.rs

```diff
@@ -124,12 +142,28 @@ impl Token {
         }
         let has_previous_page = min_index > 0;
 
+        let total_count: i32 = sqlx::query_scalar!(
+            "SELECT
+                MAX(index_per_token)
```

Comment:
I think this is off by one, since the `index_per_token` starts from `0`.
```suggestion
                MAX(index_per_token) + 1
```